modules/allpass.h

```diff
@@ -0,0 +1,63 @@
+/*  Allpass
+    Allpass filter module
+  
+    Passes all frequencies at their original levels, with a phase shift.
+  
+    Original author(s) : Barry Vercoe, John ffitch
+    Year : 1991
+    Ported from soundpipe by Ben Sergentanis, May 2020
+*/
```

Comment:
should be `/** Allpass filter module`, and should move down to line22 where current start of class is. No need to have the class name as Doxygen will grab that from the class definition.